tools/deb/node-agent-start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+# Script to configure and start the Istio node agent.
+
+set -e
+
+./nodeagent --ca-address=istio-ca:8060 --env=0 --alsologtostder --service-identity=spiffe://cluster.local/ns/default/sa/default
```

Comment:
Fully qualified name (istio-ca.default.svc....).

If you use script:
- 'exec' ( to make systemd happy )
- if you need to allow customization - you may need to source the env file. It looks you don't.

Reply:
Fixed, please take a look. THanks.